app/Controller/PagesController.php

```diff
@@ -36,6 +37,49 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
+    public function foo(  ){
```

Comment:
__[code_sniffer]__
* Expected 0 spaces between brackets of function declaration; 2 found
* Double space found
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->